src/util/Fetcher.ts

```diff
@@ -32,6 +32,9 @@ export interface IFetcher {
 export default class Fetcher implements IFetcher {
   async fetch(url: RequestInfo | URL, init?: RequestInit): Promise<Response> {
     const fetchUrl = url instanceof URL ? url.toString() : url;
```

Comment:
Also, who are we doing this `toString` for? Is that also cross-fetch?

Reply:
The standard `fetch` signature does not include the `URL` type, which is accepted here, so I assume that's the motivation.